### DISCUSSION TITLE
マージンを変更する方法は？

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1841] [Q&A] [answered]
マージンを調整するためにLaTexを使用する必要がありますか？

![image](https://user-images.githubusercontent.com/49851974/180943089-4bf4b354-ee8c-4de3-98ef-eb5c70c6a0e5.png)
![image](https://user-images.githubusercontent.com/49851974/180943535-7237a264-8e07-4b51-a484-b11bc6d89ed4.png)

**Comment by kmuto** [comment, marked as answer]
config.ymlのtexdocumentclassの「`line_length=行の文字数zw`」、「`gutter=内側スペースmm`」で調整はできます。
横に広げるとはすなわち、行の文字数を増やし、見開きでの内側スペースを狭くする ということになります。

```
texdocumentclass: ["review-jsbook", "media=print,paper=a5,line_length=42zw,gutter=5.8mm"]
```

- デフォルトのmedia=printは印刷用なので、電子的に配布するPDFであればmedia=ebookにします。
- 印刷用、かつ製本する場合は、左右には必ず一定の余白が必要です。
- マージン調整については、`review-init -w` でのGUI画面で調整するのがよいでしょう。既存のプロジェクトに対しての再実行はできないので、`review-init -w dummy1` など適当に実行し、生成されたconfig.ymlのtexdocumentclassの値を本番のプロジェクトにコピーします。

**Reply by jasondee1992** [reply]
TOPとBottomからのマージンはどうですか？

**Reply by kmuto** [reply]
同様に`number_of_lines` (行数) と `head_space` (上からのスペース) で調整します。`review-inti -w` の結果で確認するのがよいでしょう。

**Reply by jasondee1992** [reply]
作成しようとしましたが、エラーが発生しました。

![image](https://user-images.githubusercontent.com/49851974/181194629-f81f7891-c85e-4de8-8b95-21236a09c3b3.png)

![image](https://user-images.githubusercontent.com/49851974/181195743-e1ad9e05-b0e6-438f-b9f2-3a73b52c16d0.png)

Directory
![image](https://user-images.githubusercontent.com/49851974/181194734-469825b8-6eaf-4d55-98b7-5b88acb8663a.png)

**Reply by kmuto** [reply]
> 既存のプロジェクトに対しての再実行はできないので、review-init -w dummy1 など適当に実行し、生成されたconfig.ymlのtexdocumentclassの値を本番のプロジェクトにコピーします。

と書いたとおり、あくまでもダミーのプロジェクト名を付けて別途作成し、そのconfig.ymlのtexdocumentclassの結果をformat_guideプロジェクトのほうのconfig.ymlに転記しましょう。

**Reply by jasondee1992** [reply]
kmutoさんありがとうございます。
